package.json

```diff
@@ -8,7 +8,8 @@
     "typings"
   ],
   "scripts": {
-    "build": "tsc && npm run copy-static",
+    "build": "tsc && npm run copy-static && npm run build-bundle -- --minify",
+    "build-bundle": "esbuild ./src/bundle/index.ts --outdir=./build/src/bundle --bundle --format=cjs --platform=node --target=ES2021",
```

Comment:
Here I build a bundle in order to not install all heavy babel deps in `dependencies`

Reply:
I think we should add it to the `watch` script as well.

In watch script, we can either build the bundle once on startup, or make 2 separate scripts like `watch:bundle` `watch:src` and unite them in `watch` script via `concurrently` (we use the [same approach](https://github.com/gemini-testing/html-reporter/blob/9d860fef7b455d9b63b4fcbfc91ad5ed38754eae/package.json#L34) in html-reporter).

I'm not sure how often we'll want to change (watch) bundle contents, so it's up to you to decide.